sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs

```diff
@@ -89,24 +88,36 @@ protected ServiceBusClient()
         }
 
         /// <summary>
-        ///
+        /// The connection that is used for the client.
         /// </summary>
         internal ServiceBusConnection Connection { get; }
 
         /// <summary>
-        ///
+        /// Initializes a new instance of the <see cref="ServiceBusClient"/> class.
         /// </summary>
-        /// <param name="connectionString"></param>
+        /// <param name="connectionString">The connection string to use for connecting to the
+        /// Service Bus namespace.
+        /// </param>
+        /// <remarks>
+        /// If the connection string specifies a specific entity name, any subsequent calls to
+        /// GetSender/GetReceiver/etc., must specify the same entity name.
+        /// </remarks>
```

Comment:
Can we use the same format for docs as other classes? like skip one line after summary, param ends.
ex:
```csharp
  /// <summary>
  ///  Initializes a new instance of the <see cref="ServiceBusClient"/> class.
  /// </summary>

  /// <param name="connectionString">The connection string to use for connecting to the Service Bus 
  /// namespace.</param>

  /// <remarks>
  ///  If the connection string specifies a specific entity name, any subsequent calls 
  ///  GetSender/GetReceiver/etc., must specify the same entity name.
  /// </remarks>
```